project/assets/login.less

```diff
@@ -0,0 +1,117 @@
+body {
+    background-color: rgb(82, 86, 89);
+    margin-top: 50px;
+    margin-left: 0;
+}
+
+/* First section, including the 'feedback' background image and the login form */
+
+.login-form {
+    opacity: 0.8;
+}
+
+.login-form-background-image {
+    width: 1897px;
+    opacity: 0.8;
```

Comment:
Please remove this declaration(opacity: ... ) and apply the login-form class to the element